recipe/meta.yaml

```diff
@@ -21,9 +21,9 @@ requirements:
     - setuptools
     - wheel
   run:
-    - python >=3.5
+    - python
     - gitdb >=4.0.1,<5
-    - typing-extensions >=3.7.4.0
+    - typing-extensions >=3.7.4.0  # [py<38]
```

Comment:
Must be >=3.7.4.**3**, see https://github.com/gitpython-developers/GitPython/blob/141cd651e459bff8919798b3ccf03dfa167757f6/requirements.txt#L2

Reply:
Fixed. Thanks!